cmd/renterd/main.go

```diff
@@ -148,6 +148,37 @@ func main() {
 		node.AutopilotConfig
 	}
 	autopilotCfg.RevisionSubmissionBuffer = api.BlocksPerDay
+
+	// TODO: the following flags will be deprecated in v1.0.0 in favor of
+	// environment variables to ensure we do not ask the user to pass sensitive
+	// information via CLI parameters.
+	flag.StringVar(&dbCfg.password, "db.password", "", "password for the database to use for the bus - can be overwritten using RENTERD_DB_PASSWORD environment variable")
+	flag.StringVar(&busCfg.apiPassword, "bus.apiPassword", "", "API password for remote bus service - can be overwritten using RENTERD_BUS_API_PASSWORD environment variable")
+	flag.StringVar(&busCfg.remoteAddr, "bus.remoteAddr", "", "URL of remote bus service - can be overwritten using RENTERD_BUS_REMOTE_ADDR environment variable")
+	flag.StringVar(&workerCfg.apiPassword, "worker.apiPassword", "", "API password for remote worker service")
+	flag.StringVar(&workerCfg.remoteAddrs, "worker.remoteAddrs", "", "URL of remote worker service(s). Multiple addresses can be provided by separating them with a semicolon. Can be overwritten using the RENTERD_WORKER_REMOTE_ADDRS environment variable")
+
+	for _, flag := range []struct {
```

Comment:
Won't this loop print all the warning every time? Instead of just printing it when a flag is actually used?

Reply:
there's `if flag.input != "" {` to make sure that doesn't happen, they all default to `""`